projects/kit/components/calendar-month/calendar-month.component.ts

```diff
@@ -39,8 +37,13 @@ export class TuiCalendarMonthComponent implements TuiWithOptionalMinMax<TuiMonth
     public year: TuiYear = TODAY;
 
     @Input()
-    public disabledItemHandler: TuiBooleanHandlerWithContext<TuiMonth, TuiMonthContext> =
-        TUI_FALSE_HANDLER;
+    public disabledItemHandler: TuiBooleanHandler<TuiMonth> = TUI_FALSE_HANDLER;
+
+    @Input()
+    public minLength: number | null = null;
+
+    @Input()
+    public maxLength: number | null = null;
```

Comment:
shouldn't we add new inputs to the `CalendarMonth` API documentation as well?

Reply:
Forgot we had it ) Added